backend/src/v2/driver/driver.go

```diff
@@ -940,3 +1021,203 @@ func generateOutputURI(root, artifactName string, taskName string) string {
+	// Can only provide at most one of these two parameters.
+	// If neither is provided, PVC name is a random generated UUID.
+	pvcNameSuffixInput := inputs.ParameterValues["pvc_name_suffix"]
+	pvcNameInput := inputs.ParameterValues["pvc_name"]
+	if pvcNameInput.GetStringValue() != "" && pvcNameSuffixInput.GetStringValue() != "" {
+		return "", fmt.Errorf("failed to create pvc: at most one of pvc_name and pvc_name_suffix can be non-empty")
+	} else if pvcNameSuffixInput.GetStringValue() != "" {
+		pvcName = uuid.NewString() + pvcNameSuffixInput.GetStringValue()
+	} else if pvcNameInput.GetStringValue() != "" {
+		pvcName = pvcNameInput.GetStringValue()
+	} else {
+		pvcName = uuid.NewString()
+	}
+
+	// Required input: size
+	volumeSizeInput, ok := inputs.ParameterValues["size"]
+	if !ok {
+		return "", fmt.Errorf("failed to create pvc: parameter volumeSize not provided")
+	}
+
+	// Optional input: storage_class_name
+	// When not provided, use default value `standard`
+	storageClassNameInput, ok := inputs.ParameterValues["storage_class_name"]
+	var storageClassName string
+	if !ok {
+		storageClassName = "standard"
+	} else {
+		storageClassName = storageClassNameInput.GetStringValue()
+	}
+
+	// Optional input: annotations
+	pvcAnnotationsInput := inputs.ParameterValues["annotations"]
+	pvcAnnotations := make(map[string]string)
+	for key, val := range pvcAnnotationsInput.GetStructValue().AsMap() {
+		typedVal := val.(structpb.Value)
+		pvcAnnotations[key] = typedVal.GetStringValue()
+	}
+
+	// Optional input: volume_name
+	volumeNameInput := inputs.ParameterValues["volume_name"]
+	volumeName := volumeNameInput.GetStringValue()
+
+	// Initialize Kubernetes client set
+	restConfig, err := rest.InClusterConfig()
+	if err != nil {
+		return "", fmt.Errorf("failed to initialize kubernetes client: cannot get incluster config: %w", err)
+	}
+	k8sClient, err := kubernetes.NewForConfig(restConfig)
+	if err != nil {
+		return "", fmt.Errorf("failed to initialize kubernetes client: %w", err)
+	}
+
+	// Create a PersistentVolumeClaim object
+	pvc := &corev1.PersistentVolumeClaim{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:        pvcName,
+			Annotations: pvcAnnotations,
+		},
+		Spec: corev1.PersistentVolumeClaimSpec{
+			AccessModes: accessModes,
+			Resources: corev1.ResourceRequirements{
+				Requests: corev1.ResourceList{
+					corev1.ResourceStorage: resource.MustParse(volumeSizeInput.GetStringValue()),
+				},
+			},
+			StorageClassName: &storageClassName,
+			VolumeName:       volumeName,
+		},
+	}
+
+	// Create the PVC in the cluster
+	createdPVC, err := k8sClient.CoreV1().PersistentVolumeClaims(namespace).Create(context.Background(), pvc, metav1.CreateOptions{})
+	if err != nil {
+		return "", fmt.Errorf("failed to create pvc: %w", err)
+	}
+	glog.Infof("Created PVC %s\n", createdPVC.ObjectMeta.Name)
+	return createdPVC.ObjectMeta.Name, nil
+}
+
+func DeletePVC(inputs *pipelinespec.ExecutorInput_Inputs, namespace string) error {
```

Comment:
NIT: add a comment describing the function

Reply:
Same as above, adding comments describing this function feels quite redundant.